package.json

```diff
@@ -2,7 +2,7 @@
   "name": "apidocs",
   "version": "0.1.0",
   "private": true,
-  "homepage": "apidocs",
+  "homepage": ".",
```

Comment:
won't this break the deployment? 
react uses the `homepage` thing to setup the base-url for all the assets.  
Using `.` will make the assets relative to the `index.html`

Reply:
Removed this change :+1: